includes/class-satispress-package-plugin.php

```diff
@@ -218,9 +218,72 @@ public function get_version() {
 	 */
 	protected function get_data( $prop ) {
 		if ( empty( $this->data ) ) {
-			$this->data = get_plugin_data( $this->get_file(), false, false );
+			$this->data = $this->get_plugin_data( $this->get_file(), false, false );
 		}
 
 		return ( isset( $this->data[ $prop ] ) ) ? $this->data[ $prop ] : '';
 	}
+
+
+	/**
+	 * Get the plugin details
+	 *
+	 * This function is a copy of the original WordPress core function, with "Plugin Type"
```

Comment:
Can you please highlight which lines have changed from the original function? I can't spot any differences.